README.md

```diff
@@ -34,14 +34,28 @@ const bytesWritten = await zipper.finalize()
 // bytesWritten -> 276
 ```
 
-## Testing
+## Options
+
+Options can be passed as the second argument to `addDir`:
+
+```js
+await zipper.addDir('directory', {
+  ignoreHidden: true,
+  exclude: ['/tmp/']
+})
+```
 
-Run `npm test` to run a simple test suite; this tests that no errors occur, but
-it does not test that the file is created correctly. Please manually inspect
-`test/output.zip` and ensure it matches the files in `test/fixtures`.
+`ignoreHidden`: whether to ignore hidden files (default: `false`)
+
+`exclude`: array of paths to exclude (default: `[]`)
```

Comment:
Mention that it will also exclude files if they match the pattern.

Reply:
Done :+1: